includes/connect/class-wc-stripe-connect-api.php

```diff
@@ -125,13 +125,27 @@ protected function request( $method, $path, $body = [] ) {
 			}
 
 			$url = trailingslashit( WOOCOMMERCE_CONNECT_SERVER_URL );
-			$url = apply_filters( 'wc_connect_server_url', $url );
+			$url = apply_filters_deprecated(
+				'wc_connect_server_url',
+				[ $url ],
+				'9.7.0',
```

Comment:
Perhaps we should set the version to 9.6.0 as this is slated to go out in the next release?